gulp-tasks/breakpoints.js

```diff
@@ -13,7 +13,7 @@ const themeName = options.theme.name;
 const {parallel} = require('gulp');
 const WARNING = 'THIS IS GENERATED FILE. PLEASE EDIT THEMENAME.breakpoints.yml';
 const jsFile = '.bp.json';
-const sassFile = '_bp.scss';
+const sassFile = 'abstractions/_bp.scss';
```

Comment:
maybe better to add smth like options.theme.variables ?

Reply:
We have other files in abstractions folder, so other must be in options.theme.variables as well. I don't think we need it.